charts/istio-alerts/templates/service-prometheusrule.yaml

```diff
@@ -32,4 +32,27 @@ spec:
         {{- end }}
     {{- end }}
     {{- end }}
+
+     # Adding latency alarm to trigger when request latency is above > threshold
+    {{ with .Values.serviceRules.highRequestLatency }}
+    - alert: HighRequestLatency
+      annotations:
+        summary: Processes experience latency above {{ .threshold }}.
+        runbook_url: {{ $values.defaults.runbookUrl}} #HighRequestLatency
+        description: >-
+          {{`{{ $value | humanizePercentage }} latency is above
+          namespace {{ $labels.namespace }} for container {{ $labels.container
+          }} in pod {{ $labels.pod }}.`}}
+      expr: |-
+        sum by (destination_service_name) (irate(istio_request_duration_milliseconds_bucket{reporter=~"destination",destination_service_namespace="{{ $release.Namespace }}"}[1m])) by (le)) / 1000)
```

Comment:
without the `by (le)` and some paren adjustment it looks like this query works: 

https://prometheus.staging-us1-eks.nextdoor-test.com/graph?g0.expr=sum%20by%20(destination_service_name)%20(irate(istio_request_duration_milliseconds_bucket%7Breporter%3D~%22destination%22%2Cdestination_service_namespace%3D%22goflask%22%7D%5B1m%5D)%20%2F%201000)&g0.tab=0&g0.stacked=0&g0.show_exemplars=0&g0.range_input=1h